packages/clay-drop-down/src/DropDown.tsx

```diff
@@ -28,6 +28,13 @@ interface IProps extends React.HTMLAttributes<HTMLDivElement | HTMLLIElement> {
 	 */
 	active: boolean;
 
+	/**
+	 * Flag to align the DropDown menu within the viewport.
+	 */
+	alignmentByViewport?: React.ComponentProps<
```

Comment:
Naming is hard, I'm wondering if this shouldn't be `alignToViewport`

Reply:
Yeah, this is pretty complicated I thought of leaving `align*` but I saw that we have the `alignmentPosition` API but it also has `alignElementRef`, I got closer to the same name as the `dom-align` API which is `alwaysByViewport`. I don't have very strong arguments so I'm fine with any name.

Reply:
> ...but I saw that we have the alignmentPosition

You're right ... when reviewing this again I saw it. So I think we can keep this as it is right now.